driver-adapters/tidb-node-basic/index.test.js

```diff
@@ -0,0 +1,107 @@
+// @ts-check
+const { test, expect } = require('@jest/globals')
+const { handler } = require('./index')
+const { dependencies } = require('./package.json')
+
+jest.setTimeout(10_000)
```

Comment:
Is a custom timeout necessary?

Reply:
Copy/paste from the other driver-adapter tests 🤷

Reply:
It is necessary as we are issuing many requests one after another and that may take more than the default 5 seconds.